database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
+* go
+  
+  [Download and install go](https://go.dev/doc/install)
+* psql
+  ```bash
+  sudo dnf install postgresql
+  psql --version
+  psql (PostgreSQL) 15.6
+  ```
+* DBeaver Community
+  
+  A SQL client is recommended to interact easily with the Database. 
+  [DBeaver Community](https://dbeaver.io/).
+
+## Manual deployment
+
+The PostgreSQL manual deployment consists in 2 parts: First, the deployment of the Database infrastructure and then, using a script, create a table in the Database and populate it.
+
+### Deploy the Database infrastructure
+
+* Create a ConfigMap with the Database details.
+
+In the file `postgres-cm.yaml` add the configuration parameters for the Database in the `data` section.
+```yaml
+# ...
+data:
+  POSTGRES_DB: <db_name> # postgresdb
+  POSTGRES_USER: <db_user> # admin or ps_user
+  POSTGRES_PASSWORD: <db_pass> # a secure password
```

Comment:
We should not have a password in a config file - we need to come up with an alternative for this.

Reply:
yes, I agreed. This was the fastest way to implement this. I'll open a ticket to replace this by a secret for the manual deployment.
Also, the helm deployment will not use a CM to store the password.